posts/2023-09-06-coming-soon-gov-changes.md

```diff
@@ -9,4 +9,32 @@ tags:
   - Program updates
 ---
 
-We’re building a new way to get a .gov. This is a beta site and is for testing purposes only. Go to [get.gov to read about important changes coming to .gov this fall](https://get.gov/updates/2023/9/6/infrastructure-as-a-public-service/). 
+It should be easy for governments to get a .gov domain – and hard to manage one insecurely. So, over the next few months, CISA will initiate major changes to the .gov top-level domain’s tech infrastructure, laying a foundation for the services we will offer in the coming years. We have great plans for making .gov the place governments want to be online. 
+
+We [announced](../2022-09-14-making-infrastructure-less-invisible/) much of this work a year ago; here’s an update on our progress. 
+
+## Three changes coming to .gov this fall 
+
+1. **Moving the .gov registry and DNS**: The most critical element of our work, the *global publication of the .gov DNS zone and related registry software*, will transition to a new service provider. This shift will be carefully coordinated to ensure no impact to U.S.-based government organizations or the public, who depend on .gov websites and email addresses. (To be clear, we don’t run those websites or mail servers, but .gov is the supporting technology layer that makes them navigable online.) 
+
+2. **A new online home for .gov domains**: We’re launching a new *registrar*, the online home for governments to request and manage a .gov domain. We put the domain request process online (no more physical signatures required). The new registrar will enable many future features, like DNS hosting. Moving to the new registrar requires our users to take some actions, which we detailed in an [email to domain registrants](../2023-transition-email/). 
+
+3. **A new website for .gov**: Finally, we’re sharing a *beta version of our new website*, which will help governments learn about .gov and why they should use it. We’re still building and testing [get.gov](https://get.gov/). 
+
+Both the registrar and our new website are being developed as open source software. You can check out the code for [the registrar](https://github.com/cisagov/getgov){.usa-link--external}  and our [public site](https://github.com/cisagov/get.gov){.usa-link--external} on GitHub.
```

Comment:
Change first repo name to cisagov/manage.get.gov

Reply:
Fixed. Thank you!